internal/venice-common/src/main/java/com/linkedin/venice/utils/HelixUtils.java

```diff
@@ -381,4 +383,27 @@ public static void setupCustomizedStateConfig(HelixAdmin admin, String clusterNa
   public static String instanceIdToUrl(String instanceId) {
     return "https://" + instanceId.replace("_", ":");
   }
+
+  public static CloudConfig getCloudConfig(
+      CloudProvider cloudProvider,
+      String cloudId,
+      List<String> cloudInfoSources,
+      String cloudInfoProcessorName) {
+    CloudConfig.Builder cloudConfigBuilder =
+        new CloudConfig.Builder().setCloudEnabled(true).setCloudProvider(cloudProvider);
+
+    if (!cloudId.isEmpty()) {
```

Comment:
can we use `StringUtils.isEmpty` instead String:isEmpty here and later?

Reply:
Good suggestion